server/lease/lessor.go

```diff
@@ -572,6 +572,9 @@ func (le *lessor) Detach(id LeaseID, items []LeaseItem) error {
 
 	l := le.leaseMap[id]
 	if l == nil {
+		for _, it := range items {
+			delete(le.itemMap, it)
+		}
```

Comment:
it does that, exactly 5 lines below

Reply:
thx for review. when lease is revoked, in le.Revoke it will delete le.leaseMap , and then delete key in . when delete key, it will find one lease use le.GetLease use leaseItem. and call le.Detach. for now, the leaseMap does not contain the lease,  there is no chance to delete le.itemMap.

Reply:
Ah, there's a return before. Apologies, that makes sense.